cli/magic-create.ts

```diff
@@ -377,12 +399,24 @@ async function processCreateOptions(options: any): Promise<void> {
     models.defaultEmbedding = embeddingModels[0].name;
   }
 
-  config.rag.crossEncoderModels[0] = {
-    provider: "sagemaker",
-    name: "cross-encoder/ms-marco-MiniLM-L-12-v2",
-    default: true,
-  };
-  config.rag.embeddingsModels = embeddingModels;
+  if (answers.enableCrossEncoding && answers.enableSagemakerModels) {
```

Comment:
if `enableSagemakerModels === true` you can deploy both embeddings and cross encoder models.